lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4

```diff
@@ -32,7 +32,10 @@ streamNode
     ;
 
 acceptNode
-    : k=AcceptKeyword acceptURI=location ( AsKeyword text=Name )? (OptionKeyword TransportKeyword value=location)?
+    : k=AcceptKeyword acceptURI=location ( AsKeyword text=Name )?
+      (OptionKeyword TransportKeyword value=location)?
```

Comment:
Does this require a specific ordering for the OptionKeywords, and if so it that desired?  I would think not especially as we add more options type, but I would be alright with the ones and order we have here...